src/libpmemfile-posix/utils.h

```diff
@@ -36,6 +36,9 @@
 #include "inode.h"
 #include "layout.h"
 
+#define ASSERT_IN_TX() ASSERTeq(pmemobj_tx_stage(), TX_STAGE_WORK)
```

Comment:
Could these two be functions instead of macros? I can live with this, just asking...